stardis/model/geometry/radial1d.py

```diff
@@ -1,5 +1,7 @@
-class Radial1DGeometry:
+from tardis.io.util import HDFWriterMixin
 
+
+class Radial1DGeometry(HDFWriterMixin):
```

Comment:
I don't think we need to be as granular as creating a whole HDF for the geometry class. It's a very small class with not too much data. Maybe ~60 floats total for a typical simulation. Also, it's included in the StellarModel class, so I think if possible you could incorporate any geometry tests into that.

Reply:
Fixed in https://github.com/tardis-sn/stardis/pull/213/commits/de10eb66b6e772c3cc6fdce871224ff1425e59b6